app/src/main/java/com/teamagam/gimelgimel/app/map/esri/plugins/ScaleBar.java

```diff
@@ -95,4 +66,22 @@ private String format(double scale) {
         String formattedScale = mDecimalFormatter.format((int) scale);
         return String.format("1 : %s", String.valueOf(formattedScale));
     }
+
+    private class ScaleBarUIUpdatePoller extends UIUpdatePoller {
+
+        ScaleBarUIUpdatePoller() {
+            super(Schedulers.computation());
+        }
+
+        @Override
+        protected void periodicalAction() {
+            if (mMapView != null) {
```

Comment:
Better check it on start, so when view is not set you don't even start the poller.